build.env

```diff
@@ -21,6 +21,9 @@ GO111MODULE=on
 GOLANGCI_VERSION=v1.27.0
 GOSEC_VERSION=v2.3.0
 
+# external snapshotter version
```

Comment:
@pkalever please mention the snapshotter release page as a source comment here, that would help

Reply:
Sure Humble :+1:

Reply:
:+1: